### PR TITLE
hybrid Quickstart: Add hybrid 1.4 installation script and Cloud Build…

### PR DESCRIPTION
**Pull request by danistrebel**
What's changed, or what was fixed?

- Bump to Apigee hybrid 1.4
- Bump ASM to 1.7
- Cloud Build Config for Apigee hybrid

- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

File: tools/hybrid-quickstart/cloudbuild/hybrid.cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+steps:
+# Step 1: Install node dependencies for linting and testing
+- name: gcr.io/cloud-builders/gcloud
+  entrypoint: "bash"
+  args:
+    - "-c"
+    - |-
+      # git clone https://github.com/apigee/devrel.git
+
+      git clone https://github.com/danistrebel/devrel.git && cd devrel
```
Comment (laughingbiscuit) [inline tools/hybrid-quickstart/cloudbuild/hybrid.cloudbuild.yaml:36]: feature branch

File: tools/hybrid-quickstart/cloudbuild/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+If you run the Apigee hybrid quickstart from a workstation where you don't have
+access to the necessary tools such as kubectl or want to create a hybrid cluster
+in a GitOps pipeline, you can use this configuration that uses Google Cloud
+Build to create the hybrid runtime for you.
+
+## Prerequisites
+
+To limit the permission scope of the Cloud Build service account we create the
+required service accounts and enable all required APIS beforehand.
```
Comment (laughingbiscuit) [inline tools/hybrid-quickstart/cloudbuild/README.md:11]: (trivial) uppercase S

File: tools/hybrid-quickstart/cloudbuild/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+### Initialize project-level APIs
+
+First you need to enable all required APIs. You can use the function in the
+steps.sh file or enable them manually via the UI or API.
+
+```sh
+# from within the apigee hybrid quickstart folder
```
Comment (laughingbiscuit) [inline tools/hybrid-quickstart/cloudbuild/README.md:19]: ./tools/hybrid-quickstart directory

File: tools/hybrid-quickstart/cloudbuild/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+source steps.sh
+enable_all_apis
+```
+
+### Create service accounts
+
+You can use apigeectl to create the service accounts or create them manually via
+the UI or API according to the documentation.
```
Comment (laughingbiscuit) [inline tools/hybrid-quickstart/cloudbuild/README.md:27]: link to docs

File: tools/hybrid-quickstart/cloudbuild/README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+```sh
+gcloud services enable cloudbuild.googleapis.com
+
+PROJECT_NUMBER=$(gcloud projects describe "${PROJECT_ID}" --format="value(projectNumber)")
+
+for ROLE in container.admin container.clusterAdmin editor iam.serviceAccountKeyAdmin cloudbuild.builds.builder
+do
+gcloud projects add-iam-policy-binding "$PROJECT_ID" \
+   --member="serviceAccount:${PROJECT_NUMBER}@cloudbuild.gserviceaccount.com" \
```
Comment (laughingbiscuit) [inline tools/hybrid-quickstart/cloudbuild/README.md:58]: maybe remove braces?

File: tools/hybrid-quickstart/cloudbuild/hybrid.cloudbuild.yaml [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+      cd ..
+
+      QUICKSTART_ROOT=$$(pwd)/devrel/tools/hybrid-quickstart/hybrid14
+      source "$$QUICKSTART_ROOT/steps.sh"
+
+      set_config_params
+
+      create_apigee_org
+
+      create_apigee_env $_QUICKSTART_ENV_NAME
```
Comment (laughingbiscuit) [inline tools/hybrid-quickstart/cloudbuild/hybrid.cloudbuild.yaml:47]: quotes around vars

File: tools/hybrid-quickstart/hybrid14/steps.sh [hunk 7]
```diff
@@ -256,35 +259,29 @@ add_env_to_envgroup() {
 }
 
 configure_network() {
-    ENV_GROUP_NAME=$1
+    echo "🌐 Setup Networking"
 
-    echo "🌐 Set up Networking for $ENV_GROUP_NAME.$DNS_NAME"
+    ENV_GROUP_NAME=$1
```
Comment (laughingbiscuit) [inline tools/hybrid-quickstart/hybrid14/steps.sh:264]: quotes around var

File: tools/hybrid-quickstart/cloudbuild/hybrid.cloudbuild.yaml (hunk 1 above)
Comment (danistrebel) [reply tools/hybrid-quickstart/cloudbuild/hybrid.cloudbuild.yaml:36]: Gonna remove this and expect an invocation with the source context instead. This makes it more flexible.

**Review by laughingbiscuit — APPROVED** [review]

**Pull request merged**